src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import core.basesyntax.io.FileReader;
+import core.basesyntax.io.FileReaderImpl;
+import core.basesyntax.io.FileWriter;
+import core.basesyntax.io.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.servise.DataConverter;
+import core.basesyntax.servise.DataConverterImpl;
+import core.basesyntax.servise.OperationStrategy;
+import core.basesyntax.servise.OperationStrategyImpl;
+import core.basesyntax.servise.ReportGenerator;
+import core.basesyntax.servise.ReportGeneratorImpl;
+import core.basesyntax.servise.ShopService;
+import core.basesyntax.servise.ShopServiceImpl;
+import core.basesyntax.servise.operation.BalanceOperation;
+import core.basesyntax.servise.operation.OperationHandler;
+import core.basesyntax.servise.operation.PurchaseOperation;
+import core.basesyntax.servise.operation.ReturnOperation;
+import core.basesyntax.servise.operation.SupplyOperation;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    private static final String DIRECTORY_PATH = "./src/main/java/core/basesyntax";
```

Comment:
Avoid hardcoding file paths in your application. The file path should be passed as an argument to your application, for example, through the command line or configuration file. This will make your application more flexible and easier to configure in different environments.